requirements/_requirements.txt

```diff
@@ -10,4 +10,5 @@ redis<6.0.0
 requests<=2.31.0
 rich<=13.5.2
 shapely<=2.0.1
-supervision<1.0.0
\ No newline at end of file
+supervision<1.0.0
+pybase64<2.0.0
```

Comment:
Curious if this actually helped at all or not

Reply:
I saw about 1-2 ish FPS increase, even for the biggest images. So certainly not nothing!